clustering/web/extension/src/main/resources/org/wildfly/extension/clustering/web/LocalDescriptions.properties

```diff
@@ -36,3 +36,6 @@ distributable-web.affinity.remove=Removes an affinity configuration
 distributable-web.affinity.none=Web requests will not have an affinity for any cluster member
 distributable-web.affinity.local=Web requests will have an affinity for the member that last handled a given session
 distributable-web.affinity.primary-owner=Web requests will have an affinity for the primary owner of a given session
+distributable-web.affinity.ranked=Web requests will have an affinity for the first available node in a list containing primary and backup owners, and the member that last handled a given
```

Comment:
is the world "session" needed at the end of the phrase?

Reply:
Yes indeed.